8-algoruthm-and-data-structures-2/hometask/src/JobRunner.ts

```diff
@@ -0,0 +1,79 @@
+interface Job {
+    priority: number;
+}
+
+class JobRunner {
```

Comment:
I'd separate classes, one to run jobs, another one to manage elements with certain priority (priority queue, for instance).